miio/integrations/vacuum/roborock/vacuum.py

```diff
@@ -403,11 +403,16 @@ def manual_control(
     @command()
     def status(self) -> VacuumStatus:
         """Return status of the vacuum."""
-        status = VacuumStatus(self.send("get_status")[0])
+        status = self.vacuum_status()
         status.embed(self.consumable_status())
         status.embed(self.clean_history())
         return status
 
+    @command()
+    def vacuum_status(self) -> VacuumStatus:
+        """Return status of the vacuum."""
+        return VacuumStatus(self.send("get_status")[0])
```

Comment:
This change is separate and should be in your other PR, shouldn't it?

Reply:
I will move it